src/datomic_toolbox/core.clj

```diff
@@ -1,20 +1,31 @@
 (ns datomic-toolbox.core
   (:require [clojure.java.io :as io]
             [clojure.walk :as walk]
-            [datomic.api :as d]
-            [turbovote.resource-config :refer [config]])
+            [datomic.api :as d])
   (:refer-clojure :exclude [partition]))
 
+(def default-uri (atom nil))
+(def default-connection (atom nil))
+(def default-partition (atom nil))
+
+(defn configure! [{:keys [uri partition]}]
+  (reset! default-uri uri)
+  (reset! default-partition partition))
+
+(defn uri [] @default-uri)
+
 (defn connection []
-  (d/connect (config [:datomic :uri])))
+  (let [conn (d/connect (uri))]
+    (reset! default-connection conn)
```

Comment:
@ericnormand It's strictly a backwards compatibility thing. The master branch version pulls the connection info from the resource-config state every time, so this version saves it in an atom instead. It gets used in 4 places in the code:

1. https://github.com/democracyworks/datomic-toolbox#L27
1. https://github.com/democracyworks/datomic-toolbox#L88
1. https://github.com/democracyworks/datomic-toolbox#L96
1. https://github.com/democracyworks/datomic-toolbox#L101